independent-publisher-connectors/AzureADExtensions/apiDefinition.swagger.json

```diff
@@ -0,0 +1,504 @@
+{

```

Comment:
@paulculmsee Please let us know the reason to add `AzureAdExtensions` connector files where the PR title is for `PowerBI` and `Jira` conectors.

Reply:
My mistake - still learning and will remove